shell/assets/translations/en-us.yaml

```diff
@@ -6483,7 +6491,7 @@ model:
       okta: Okta
       freeipa: FreeIPA
       googleoauth: Google
-      oidc: OIDC
+      oidc: Generic OIDC
```

Comment:
Problem compounds. `Generic OIDC` looks good when selecting it as a provider, however looks bad when logging in `Log in with Generic OIDC`. Can the log in button still remain as `Log in with OIDC`? Should be possible via the log in component.

Apologies for the churn.

Reply:
No worries, I updated the login button to use `OIDC` as the provider display name:

![oidclogin](https://github.com/rancher/dashboard/assets/40806497/4006aaa0-289a-40b6-ba0d-c86782bb8ad6)